src/components/Photo/DocumentAutoCapture.js

```diff
@@ -105,6 +106,7 @@ export default class DocumentAutoCapture extends Component<Props, State> {
     return (
       <Camera
         {...this.props}
+        className={style['docAutoCaptureFrame']}
```

Comment:
[Minor] Probably your linter does not pick this up, but should better be `style.docAutoCaptureFrame`.

Reply:
oh yeah. wasn't picked up by linter. thanks!